000-local-env.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+if ( defined( 'VIP_GO_ENV' ) && VIP_GO_ENV ) {
```

Comment:
@nickdaugherty https://github.com/Automattic/vip-go-mu-plugins/blob/master/000-vip-init.php#L6-L8

Which I'm now realizing will load after this...